massa-serialization/src/lib.rs

```diff
@@ -392,3 +395,87 @@ impl Deserializer<bool> for BoolDeserializer {
         })(buffer)
     }
 }
+
+/// Serializer for Ratio
```

Comment:
why do we need to serialize ratios ? I don't think we should be sending them through the wire

Reply:
Let's open a followup on this

Reply:
https://github.com/massalabs/massa/issues/4183